components/side-drawer/src/vwc-side-drawer.scss

```diff
@@ -30,23 +32,17 @@ $inline-size: var(#{variables.$side-drawer-inline-size}, 280px);
 	color: var(#{variables.$side-drawer-color}, var(#{scheme-variables.$vvd-color-on-canvas}));
 	inline-size: #{$inline-size};
 
+	transform: translateX(0%);
+
 	&.side-drawer-alternate {
 		background-color: var(#{scheme-variables.$vvd-color-canvas});
 	}
-
-	&.side-drawer-modal,
-	&.side-drawer-dismissible {
-		transform: translateX(0%);
-		&:not(.side-drawer-open) {
-			&.side-drawer-end {
-				transform: translateX(100%);
-			}
-			&:not(.side-drawer-end) {
-				transform: translateX(-100%);
-			}
+	&:not(.side-drawer-open) {
+		&.side-drawer-end {
+			transform: translateX(100%);
 		}
-		@media (prefers-reduced-motion: no-preference) {
-			transition: transform 0.2s cubic-bezier(0.4, 0, 0.2, 1);
+		&:not(.side-drawer-end) {
+			transform: translateX(-100%);
 		}
 	}
 
```

Comment:
@yinonov @rachelbt Perhaps we can refactor the lines here since the default type is now dismissible.